WordPress/WordPressIntents/en.lproj/Sites.strings

```diff
@@ -10,5 +10,8 @@
 /* This text is used when the user is configuring the iOS widget to suggest them to select the site to configure the widget for */
 "gpCwrM" = "Select Site";
 
+/* This text is required by Apple to be part of the tranlsations but is not shown to the users. You can ignore translating this. */
```

Comment:
Fixing small typo , + I fear that "you can skip translating this" might suggest the translators not to submit any translation _at all_ for that key in GlotPress (leading to it being left in _untranslated_ in GlotPress and thus still left missing in the final `Sites.strings`) instead of submitting one with same copy as original. (I'm not super satisfied with my wording to explain that in a clear enough way for translators though so feel free to suggest a different wording that would explain it better)

```suggestion
/* This text is required by Apple to be part of the translations but is not shown to the users. You can keep the same verbatim English copy as the original for your submitted translation of this string instead of actually translating it in your locale. */
```